contrib/examples/actions/workflows/customfilters/mistral-json_escape.yaml

```diff
@@ -0,0 +1,17 @@
+version: '2.0'
+
+examples.mistral-customfilters-json_escape:
+    description: Example for using the custom filter "json_escape"
+    type: direct
+    input:
+        - input_str
+    output:
+        result_jinja: <% $.result_jinja %>
+        result_yaql: <% $.result_yaql %>
+    tasks:
+
+        task2:
+            action: std.noop
+            publish:
+                result_jinja: '[{"title": "{{ _.input_str|json_escape }}"}]'
```

Comment:
Why do you need to pipe _.input_str? You can't pass it directly into json_escape?